src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -125,13 +125,13 @@ pyOpenSSL==19.0.0
 python-dateutil==2.8.0
 pytz==2019.1
 requests-oauthlib==1.2.0
-requests==2.22.0
+requests==2.25.1
```

Comment:
As `azure-cli-core/setup.py` doesn't pin `requests`,

https://github.com/Azure/azure-cli/blob/f50cf6b3d63f4cdf45afc783c172394f2b990fd0/src/azure-cli-core/setup.py#L60

we can bump it at the same time.